packages/phaserx/package.json

```diff
@@ -25,7 +25,6 @@
   "dependencies": {
     "@latticexyz/utils": "workspace:*",
     "@use-gesture/vanilla": "10.2.9",
-    "mobx": "^6.7.0",
```

Comment:
Since this package is not really maintained (and should probably be considered deprecated), I am wondering if we can get away with not changing this here.

Reply:
Okay, I'll take the changes away